tidb-in-kubernetes/local-pv-management.md

```diff
@@ -0,0 +1,18 @@
+---
```

Comment:
Generally, for a how-to document that users can follow it to do some operations, we use a tile or file name that is composed of verbs if possible. This is the optimized naming rule for SEO. So I suggest changing this filename to `manage-local-pv.md`.

Reply:
Got it!